data/chl-summer-2019-ECHO.csv

```diff
@@ -0,0 +1 @@
153,VB,8,7,9,2019,37.6,27.1,21.2,97.8
154,VB,8,7,9,2019,34.7,24.3,17.8,97.8
154,VB,8,7,9,2019,21.1,10.8,4.2,97.7
154,VB,8,7,9,2019,15.1,NA,NA,97.7
156,SS,8,7,9,2019,34.2,23.8,17.3,97.6
156,SS,8,7,9,2019,31.2,20.8,14,97.4
156,SS,8,7,9,2019,38.6,28.1,22.3,97.4
156,VB,8,7,9,2019,19.5,NA,NA,97.3
156,VB,8,7,9,2019,29.9,19.5,12.7,97.3
156,VB,8,7,9,2019,27.1,16.7,9.8,97.1
157,SS,8,7,9,2019,30.7,20.3,13.5,96.7
157,SS,8,7,9,2019,26.9,16.5,9.6,96.6
157,SS,8,7,9,2019,35.6,25.1,18.8,96.6
158,VB,8,7,9,2019,37.1,26.6,20.6,96.3
158,VB,8,7,9,2019,37.3,26.8,20.8,96.2
158,VB,8,7,9,2019,36.8,26.3,20.2,96.1
159,SS,8,7,9,2019,32.3,21.9,15.2,96.1
159,SS,8,7,9,2019,33.4,23,16.4,96.1
159,SS,8,7,9,2019,34.8,24.4,17.9,96
159,VB,8,7,9,2019,29.7,19.3,12.5,95.9
159,VB,8,7,9,2019,33.9,23.5,16.9,95.7
159,VB,8,7,9,2019,23.5,13.2,6.4,95.8
161,SS,8,7,9,2019,30.8,20.4,13.6,95.6
161,SS,8,7,9,2019,27,16.6,9.7,95.6
161,SS,8,7,9,2019,28.3,17.9,11,95.6
162,VB,8,7,9,2019,36.3,25.8,19.6,95.5
162,VB,8,7,9,2019,32.7,22.3,15.6,95.3
162,VB,8,7,9,2019,27,16.6,9.7,95.3
163,VB,8,7,9,2019,52,41.4,39.8,95
163,VB,8,7,9,2019,41.4,30.9,25.7,94.9
163,VB,8,7,9,2019,39.9,29.4,23.9,94.9
165,SS,8,7,9,2019,30.8,20.4,13.6,94.7
165,SS,8,7,9,2019,30.6,20.2,13.4,94.5
165,SS,8,7,9,2019,27.5,17.1,10.2,94.5
166,VB,8,7,9,2019,35.3,24.8,18.5,94.3
166,VB,8,7,9,2019,38.2,27.7,21.9,94.3
166,VB,8,7,9,2019,38.3,27.8,22,94.2
167,SS,8,7,9,2019,30.4,20,13.2,94.2
167,SS,8,7,9,2019,27.5,17.1,10.2,94.1
167,SS,8,7,9,2019,31.3,20.9,14.1,94
167,VB,8,7,9,2019,24.4,14.1,7.2,94
167,VB,8,7,9,2019,26.1,15.7,8.9,93.8
167,VB,8,7,9,2019,33.5,23.1,16.5,93.8
168,SS,8,7,9,2019,31.7,21.3,14.6,93.6
168,SS,8,7,9,2019,34.9,24.5,18.1,93.7
168,SS,8,7,9,2019,32.1,21.7,15,93.6
170,SS,8,7,9,2019,11.5,NA,NA,93.4
170,SS,8,7,9,2019,28.9,18.5,11.6,93.3
170,SS,8,7,9,2019,29,18.6,11.7,93.3
171,VB,8,7,9,2019,26.1,15.7,8.9,93.1
171,VB,8,7,9,2019,16.2,NA,NA,93.1
171,VB,8,7,9,2019,25.5,15.1,8.3,93.1
172,SS,8,7,9,2019,32.1,21.7,15,93.1
172,SS,8,7,9,2019,36.8,26.3,20.2,92.9
172,SS,8,7,9,2019,36.2,25.7,19.5,93.1
172,VB,8,7,9,2019,33.7,23.3,16.7,92.9
172,VB,8,7,9,2019,36.9,26.4,20.3,92.8
172,VB,8,7,9,2019,35.6,25.1,18.8,92.8
173,VB,8,7,9,2019,39.1,28.6,22.9,92.8
173,VB,8,7,9,2019,40.4,29.9,24.5,92.7
173,VB,8,7,9,2019,27.5,17.1,10.2,92.7
174,SS,8,7,9,2019,34.1,23.7,17.2,92.7
174,SS,8,7,9,2019,24.4,14.1,7.2,92.7
174,SS,8,7,9,2019,34.1,23.7,17.2,92.7
175,SS,8,7,9,2019,32.5,22.1,15.4,92.5
175,SS,8,7,9,2019,40.3,29.8,24.4,92.5
175,SS,8,7,9,2019,43,32.5,27.7,92.3
175,VB,8,7,9,2019,44.3,33.8,29.4,92.2
175,VB,8,7,9,2019,40.6,30.1,24.7,92.1
176,VB,8,7,9,2019,37.4,26.9,20.9,92.1
176,VB,8,7,9,2019,37.5,27,21,92.2
176,VB,8,7,9,2019,35,24.5,18.2,92
178,SS,8,7,9,2019,30.8,20.4,13.6,91.8
178,SS,8,7,9,2019,21.8,11.5,4.8,91.8
178,SS,8,7,9,2019,29.1,18.7,11.8,91.8
179,SS,8,7,9,2019,30.6,20.2,13.4,91.8
179,SS,8,7,9,2019,20.2,9.9,3.4,91.8
179,SS,8,7,9,2019,31.9,21.5,14.8,91.7
180,VB,8,7,9,2019,27.8,17.4,10.5,91.6
180,VB,8,7,9,2019,46.1,35.5,31.7,91.5

```

Comment:
This file came out as only one line.

Reply:
Fixed. apparently the csv that get's downloaded from the CHL metering device needed re-exporting to a csv. Good to know!